src/radical/pilot/agent/agent_0.py

```diff
@@ -374,25 +373,26 @@ def _start_sub_agents(self):
                 #        out for the moment, which will make this unable to
                 #        work with a number of launch methods.  Can the
                 #        offset computation be moved to the ResourceManager?
-                bs_name = "%s/bootstrap_2.sh" % (self._pwd)
-                ls_name = "%s/%s.sh" % (self._pwd, sa)
+                bs_name       = "%s/bootstrap_2.sh" % (self._pwd)
+                launch_script = "%s/%s.launch.sh"   % (self._pwd, sa)
+                exec_script   = "%s/%s.exec.sh"     % (self._pwd, sa)
                 slots = {
                     'cpu_processes'    : 1,
                     'cpu_threads'      : 1,
                     'gpu_processes'    : 0,
                     'gpu_threads'      : 0,
-                  # 'nodes'            : [[node[0], node[1], [[0]], []]],
-                    'nodes'            : [{'name'    : node[0],
-                                           'uid'     : node[1],
+                  # 'ranks'            : [[node[0], node[1], [[0]], []]],
+                    'ranks'            : [{'node'    : node[0],
+                                           'node_id' : node[1],
```

Comment:
to check: `node_id` or `node_uid`? (if the second one then there are several files where it should be updated)

Reply:
`node_id` (for no particular reason).  I made the name more consistent in the code now.